packages/augur-sdk/src/state/getter/Trading.ts

```diff
@@ -297,8 +299,11 @@ export class Trading {
       const sharesEscrowed = convertOnChainAmountToDisplayAmount(new BigNumber(orderEventDoc.sharesEscrowed, 16), tickSize).toString(10);
       const tokensEscrowed = new BigNumber(orderEventDoc.tokensEscrowed, 16).dividedBy(10 ** 18).toString(10);
       let orderState = OrderState.OPEN;
-      if (amount === "0") {
-        orderState = orderEventDoc.eventType == 1 ? OrderState.CANCELED : OrderState.FILLED;
+      if (orderEventDoc.eventType == 3) {
```

Comment:
Should this be an enum?

Reply:
Yep, it'd be better to use OrderEventType.Fill from augur-sdk/src/state/logs/types.ts here.

Reply:
@adrake33 can you go ahead and make that change on the PR?

Reply:
Sure, will do